openshift/README.md

```diff
@@ -43,16 +43,16 @@ Depending on your needs and the [security constraints][3] of your cluster, three
 
 | Security Context Constraints   | [Restricted](#restricted-scc-operations) | [Host network](#host) | [Custom](#custom-datadog-scc-for-all-features) |
 |--------------------------------|------------------------------------------|-----------------------|------------------------------------------------|
-| Kubernetes layer monitoring    | ✅                                       | ✅                     | ✅                                             |
-| Kubernetes-based Autodiscovery | ✅                                       | ✅                     | ✅                                             |
-| Dogstatsd intake               | 🔶                                       | ✅                     | ✅                                             |
-| APM trace intake               | 🔶                                       | ✅                     | ✅                                             |
-| Logs network intake            | 🔶                                       | ✅                     | ✅                                             |
-| Host network metrics           | ❌                                       | ❌                     | ✅                                             |
-| Docker layer monitoring        | ❌                                       | ❌                     | ✅                                             |
-| Container logs collection      | ❌                                       | ❌                     | ✅                                             |
-| Live Container monitoring      | ❌                                       | ❌                     | ✅                                             |
-| Live Process monitoring        | ❌                                       | ❌                     | ✅                                             |
+| Kubernetes layer monitoring    | Supported                                | Supported                     | Supported                                             |
+| Kubernetes-based Autodiscovery | Supported                                | Supported                     | Supported                                             |
+| Dogstatsd intake               | Not Supported                            | Supported                     | Supported                                             |
+| APM trace intake               | Not Supported                            | Supported                     | Supported                                             |
+| Logs network intake            | Not Supported                            | Supported                     | Supported                                             |
```

Comment:
```suggestion
| Logs network intake            | Partially supported                      | Supported                     | Supported                                             |
```

Reply:
Hey @apigirl , I spoke with the containers team and they said the diamond meant `hostPort is not possible` and can just be listed as `Not supported`.